config/locales/views.en.yml

```diff
@@ -188,3 +188,55 @@ en:
       state:
         saving: Saving
         saved: Saved
+  imports:
+    cases:
+      button:
+        download: Download and reference example Case CSV file
+        import: Import Cases CSV
+      upload_title: Upload your CSV file
+      instruction:
+        click: Click the choose file button and navigate to the saved file and select it.
+        do_not: Do not
+        change_values: change any of the values in the first line of the example csv file.
+        import: Then click the "Import Cases CSV" button to import your cases.
+      disable_with_text: "<i class='fa fa-spinner fa-spin'></i> Importing File"
```

Comment:
I would rather not have html in this file because it is not part of what will need to be translated to another language someday

Reply:
I'll move it out :D